provider/resources.go

```diff
@@ -234,9 +233,8 @@ func Provider() tfbridge.ProviderInfo {
 		},
 	}
 
-	err := x.ComputeDefaults(&prov, x.TokensSingleModule("sumologic_",
-		mainMod, x.MakeStandardToken(mainPkg)))
-	contract.AssertNoErrorf(err, "failed to compute defaults")
+	prov.MustComputeTokens(tfbridgetokens.SingleModule("sumologic_",
+		mainMod, tfbridgetokens.MakeStandard(mainPkg)))
```

Comment:
Lets add this too.

```suggestion
	prov.MustComputeTokens(tfbridgetokens.SingleModule("sumologic_",
		mainMod, tfbridgetokens.MakeStandard(mainPkg)))
	prov.MustApplyAliases()
```

We will need to specify version for this.